### DISCUSSION TITLE
Requirements for Maryland hospitals

### DISCUSSION THREAD

**Discussion by zprice22** [CMSgov/hospital-price-transparency#17] [Q&A] [answered]
Under the Medicare waiver, Maryland hospitals do not negotiate with payers for facility services. In the new rule, hospitals are permitted to group negotiated rates for the same payer and plan type (e.g., "All PPO plans"). In the same vein, could Maryland hospitals group all payers AND plans together, rather than list all payers separately? 

Would it be acceptable to display as "All Payers" as the payer_name and "All Plans" as the plan_name?

Further, where a prompt pay discount is provided to Medicaid plans, how best should this be noted or displayed?

**Comment by shaselton-usds** [comment, marked as answer]
Please refer to the following guidance in the [Hospital Price Transparency FAQs](https://www.cms.gov/files/document/hospital-price-transparency-frequently-asked-questions.pdf):

Does the Hospital Price Transparency Final Rule apply to hospitals in the State of Maryland that are subject to global payments set by the Maryland Health Services Cost Review Commission?

Yes. If your institution meets the definition of ‘hospital’ as defined by the Hospital Price Transparency Final Rule, then your institution must comply. However, some required data elements for display may not be applicable to your hospital. For example, under the Hospital Price Transparency Rule, your hospital is obligated to make public the payer-specific negotiated charges as applicable for each item and service your hospital provides. The term “payer[1]specific negotiated charge” is defined as the charge that the hospital has negotiated with a third-party payer for an item or service. The term “third party payer” means an entity that is, by statute, contract, or agreement, legally responsible for payment of a claim for a healthcare item or service. If your hospital has not negotiated a charge with a third-party payer for an item or service your hospital provides, then your hospital would not have a “payer-specific negotiated charge” to display for that item or service.

**Reply by zprice22** [reply]
Thank you. With this guidance in mind, please see an attached example of how the CMS template may be populated as CSV Tall. Please confirm this would be acceptable. Note the comments to the far right vary by line item and scenario.
[V2.0.0_Tall_CSV_Format_Example MD hospital facility charges.csv](https://github.com/CMSgov/hospital-price-transparency/files/13864214/V2.0.0_Tall_CSV_Format_Example.MD.hospital.facility.charges.csv)

**Reply by RidenourK** [reply]
@zprice22 Thank you for your question. Please check the [Hospital Price Transparency Tools](https://cmsgov.github.io/hpt-tool/) webpage in Spring 2024 for Machine Readable File Validator 2.0. The updated validator will test machine-readable files against the new required CMS template layout and data specifications that hospitals must conform to by July 1, 2024, as outlined in the CY 2024 OPPS/ASC Final Rule.

**Reply by zprice22** [reply]
Hi - the question wasn't specific to the layout, formats and values, but more so that the scenarios are appropriately addressed. I will translate the template into statements below. Please let me know if any are not accurate. 

This is specific to facility services with Maryland hospitals and does not reflect the requirements for reporting professional fee services or any other non-regulated HSCRC services.

1. Maryland hospitals do not negotiate with payers for facility services. Rates are set by the state through a rate setting commission. 
2. Maryland hospitals can provide an explanation of the rate setting in additional_generic_notes.  They do not need to include information in the standard charge methodology or calculate estimated amounts. (See my examples in rows 4-13).
3. Maryland hospitals are not required to include charges at a payer level for facility charges, because they are not negotiated. (See my examples in rows 5-13)
4. Maryland hospitals are not required to populate additional attributes related to payer-negotiated charges for facility items and services; including, estimated amount, minimum and maximums, standard charge methodology, modifiers, payer name or plan name. (See my examples in rows 5-13)
5. Maryland hospitals are required to include all items and services that may be provided by the hospital; including, but not limited to, inpatient room charges, supplies, pharmaceuticals, unit-based charges and service packages. (See rows 4, 5, 9)
6. Maryland hospitals are required to also include the code type for those service packages and/or services that are absent a standard gross charge; including but not limited to DRG, and CPT/HCPCS abstracted from a unit-based charge capture methodology (e.g,, surgery, interventional radiology/cardiology) where the gross charges are not pre-defined but vary on the aggregation of charges. (See my example rows 5, 6, 10, 11)
7.  Maryland hospitals do not differentiate line item pricing based on outpatient vs inpatient setting. Therefore the setting for most services, except those exclusively inpatient (e.g., DRGs, inpatient rooms) or outpatient (e..g., observation, same day surgery fee), can indicate "both" as the setting. (See my example rows 4, 6, 9-13).

[V2.0.0_Tall_CSV_Format_Example MD hospital facility charges.csv](https://github.com/CMSgov/hospital-price-transparency/files/13878290/V2.0.0_Tall_CSV_Format_Example.MD.hospital.facility.charges.csv)

**Reply by RidenourK** [reply]
@zprice22 Thank you for your question. CMS is unable to comment on or provide individualized/case-by-case predeterminations as to whether a particular strategy or approach would be deemed compliant. The compliance process typically involves a comprehensive compliance review in response to a CMS audit or a complaint received through the Hospital Price Transparency website.

However, Machine Readable File Validator 2.0 will be available on the [Hospital Price Transparency Tools](https://cmsgov.github.io/hpt-tool/) webpage in Spring 2024. The updated validator will test machine-readable files against the new required CMS template layout and data specifications that hospitals must conform to by July 1, 2024, as outlined in the CY 2024 OPPS/ASC Final Rule.

**Comment by zprice22** [comment]
This is not individualized. But for the hospitals in Maryland as a whole.
As the charge capture methodology and rate setting process is regulated by
the state and standardized.

Do you agree that if there is not a payer-negotiated charge across the same
item or service that it is NOT expected that the hospital will have a
standard charge methodology? Minimum negotiated charge? Maximum negotiated
charge? Estimated allowed amount? Payer name? Plan name?
